### DISCUSSION TITLE
The theme does not render markdown strikethrough syntax correctly

### DISCUSSION THREAD

**Discussion by SWHL** [colinwilson/lotusdocs#85] [Q&A] [answered]
When I write the strikethrough syntax, the lotucs theme can't render it correctly, as shown below:
![image](https://github.com/colinwilson/lotusdocs/assets/28639377/95cb68d9-66ed-4ac4-954c-34e817f82b9c)

The source code is located in [link](https://github.com/RapidAI/RapidOCRDocs/blob/595033344fbb0c5bf832f8a408b98c665f77e693/content/docs/communicate.md?plain=1#L28). The corresponding document page address is [link](https://rapidai.github.io/RapidOCRDocs/docs/communicate/#qq%e4%ba%a4%e6%b5%81%e7%be%a4)

Appreciate any help on this.

**Comment by colinwilson** [comment, marked as answer]
You need to enclose text in [pairs of tildes](https://www.markdownguide.org/extended-syntax/#strikethrough) to enable strikethrough

**Reply by SWHL** [reply]
Thanks.
